src/pages/[platform]/build-a-backend/data/customize-authz/signed-in-user-data-access/index.mdx

```diff
@@ -50,18 +50,63 @@ const schema = a.schema({
 });
 ```
 
-## Override the authentication provider
-
-You can also override the authorization provider. In the example below, `iam` is specified as the provider which allows you to use an "Unauthenticated Role" from the Cognito identity pool for public access instead of an API key. Your Auth resources defined in `amplify/auth/resource.ts` generates scoped down IAM policies for the "Unauthenticated role" in the Cognito identity pool automatically.
```

Comment:
Same thing here, it looks like this was modified in two different PRs:
- https://github.com/aws-amplify/docs/pull/7056/files#diff-1ffd540609ef7ac5a4b4113d70fb0d7cf13a692a5eb80354369493750408c1f7
- https://github.com/aws-amplify/docs/pull/7080/files#diff-1ffd540609ef7ac5a4b4113d70fb0d7cf13a692a5eb80354369493750408c1f7

Reply:
Comparing the file on [`main`](https://github.com/aws-amplify/docs/blob/main/src/pages/gen2/build-a-backend/data/customize-authz/signed-in-user-data-access/index.mdx?plain=1) and [in this current PR](https://github.com/aws-amplify/docs/blob/1996660c7b67f73394850be8b73c86a8b28cedd0/src/pages/%5Bplatform%5D/build-a-backend/data/customize-authz/signed-in-user-data-access/index.mdx?plain=1) I actually think it is supposed to be removed? Could you give me a second set of eyes to confirm what is correct?